http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcSecurityContext.java

```diff
@@ -76,8 +76,12 @@ public String getRealm() {
     private void readObject(ObjectInputStream in) throws IOException, ClassNotFoundException {
         in.defaultReadObject();
         try {
-            token = new AccessToken(new JwtConsumerBuilder().setSkipSignatureVerification().setSkipAllValidators().build().processToClaims(tokenString));
-            idToken = new IDToken(new JwtConsumerBuilder().setSkipSignatureVerification().setSkipAllValidators().build().processToClaims(idTokenString));
+            if (tokenString != null) {
+                token = new AccessToken(new JwtConsumerBuilder().setSkipSignatureVerification().setSkipAllValidators().build().processToClaims(tokenString));
```

Comment:
Perhaps if tokenString is null, we should set token to null. Same with the case below.

Reply:
e.g.,

`token = tokenString == null ? null : new AccessToken(new JwtConsumerBuilder().setSkipSignatureVerification().setSkipAllValidators().build().processToClaims(tokenString));`

Reply:
fixed!